docs/how-to/testing-verification/test-kernel-in-proposed.md

```diff
@@ -61,10 +61,13 @@ Then install the kernel as per usual. If the kernel version in -proposed is the
 highest in any pocket, install it by running:
 
 ```{code-block} none
-sudo apt install linux-<flavour>
+sudo apt install linux-<flavour> # install kernel metapackage
```

Comment:
Hi @nwager , thanks for the additional details.
For the Ubuntu style guide, we don't encourage comments in the code block.
https://docs.ubuntu.com/styleguide/en/#code-examples-in-documentation